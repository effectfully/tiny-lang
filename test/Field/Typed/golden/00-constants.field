let ?true    = T;
let ?false   = F;
let field'   = 0;
let #vector  = { T };