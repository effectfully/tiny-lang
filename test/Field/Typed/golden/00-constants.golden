Program (Statements [ELet (UniVar {_uniVarUni = Bool, _uniVarVar = ?true_4}) (EConst (UniConst Bool True)),ELet (UniVar {_uniVarUni = Bool, _uniVarVar = ?false_5}) (EConst (UniConst Bool False)),ELet (UniVar {_uniVarUni = Field, _uniVarVar = field'_6}) (EConst 0),ELet (UniVar {_uniVarUni = Vector, _uniVarVar = #vector_7}) (EConst (UniConst Vector [True]))])